addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -293,6 +294,13 @@ export const editorCommands = {
         }
 
         currentNode = lastChildNode || currentNode;
+        if (
+            currentNode.nextSibling &&
```

Comment:
Is it impossible for `currentNode` to itself be a br? Because `<p>ab<br><br></p>` is legit.

Reply:
`currentNode` can be a `br`. Could you please elaborate the test case you are referencing to?
What should be the contentAfter here:
1. ContentBefore: `<p>[]<br></p>`,
    pasteHtml: `<p>ab<br></p>`,
    ContentAfter: ?
    
2. ContentBefore: `<p>[]<br></p>`,
    pasteHtml: `<p>ab<br><br></p>`,
    ContentAfter: ?

Reply:
1. ContentBefore: `<p>[]<br></p>`,
    pasteHtml: `<p>ab<br></p>`,
    ContentAfter: `<p>ab</p>`

2. ContentBefore: `<p>[]<br></p>`,
    pasteHtml: `<p>ab<br><br></p>`,
   ContentAfter: `<p>ab<br><br></p>`

The br is `<p>ab<br></p>` serves no function. The two brs in `<p>ab<br><br></p>` create a new line after "ab". You can try this yourself to see what it looks like.

Reply:
Hello, @Zynton After the fix in the second commit, copying and pasting `<p>ab<br></p>` from outside Odoo was working as expected it removed the last redundant `<br>`. But however copying `<p>ab<br></p>` from editor itself, the `<br>` was not removed on `insert`. I have implemented an approach to remove that `<br>` in the third commit. And about `<p>ab<br><br></p>` it works fine there is no issue with it.

PS: This `<br>` with `<p>ab<br></p>` is not related to the one we fixed in first commit.